docs/guide/content/admin.md

```diff
@@ -0,0 +1,140 @@
+# Admin Guide
+
+This section describes how the deployed application can be configured by
+setting a custom value to the config and env variables to override the
+default values. To dive into the implementation details, you can 
+explore the `ConfigurationSingleton` class in the source code.
+
+## Setting Configuration Values
+
+There are two ways to override the defaults shown below:
+
+- Create a YAML file in the directory pointed to by `LOOP_CONFIG_DIRECTORY` and set the property names as keys.
+  
+```yaml
+    download_root: "~/data/ondemand_loop"
+    max_download_file_size: "1073741824"
+    ood_dashboard_path: "/pun/sys/ood"
+```
```

Comment:
This is simply to show you that it can be done with MKdocs and for you to see how it looks

Reply:
thank you